tempalte2.yaml

```diff
@@ -1,3 +1,5 @@
 {
 "awsTokens":"sfgsdegedfgsev234"
+password:123987665
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Env Generic   
**Description:** Env Generic Credential   
**Severity:** HIGH   
**Match:** password:*********  
[This comment was created by Aqua Pipeline]